server/db/models/campaign.js

```diff
@@ -23,13 +23,9 @@ class Campaign extends BaseModel {
         },
         type: { type: 'string', enum: ['Starter', 'Accelerator', 'Grant'] },
         page_url: { type: 'string', minLength: 1 },
-        logo_url: { type: 'string', minLength: 1 },
-        letters_goal: {
-          anyOf: [{ type: 'integer', minimum: 0 }, { type: 'null' }]
-        },
-        donation_goal: {
-          anyOf: [{ type: 'integer', minimum: 0 }, { type: 'null' }]
-        }
+        campaign_tagline: { type: 'string', minLength: 1 },
```

Comment:
I think that this will need a `minLength` of 0, or to omit that property if we want to have an empty string.

Reply:
@DietBepis1 I've updated the minLength to 0. I did it this way so that when future contributors look at the code they'll see we purposefully set it to that, so that the tagline is optional.